### PR TITLE
Feature/overworld

### PR DESCRIPTION
**Pull request by 0xSense**
Purgatory level creation

### CODE REVIEW

**Review by Spencer-Trivette — APPROVED** [review]
🤘

File: data_resources/card_data/CardData.cs
Comment (PerplexerVT) [inline data_resources/card_data/CardData.cs:1]: Damage dice is being used, unsure why its being deleted here.

File: scenes/main/main.tscn
Comment (Perplexer-HNP) [inline scenes/main/main.tscn:1]: Master deck isnt being populated in the main scene?

File: scenes/combat/combat.tscn
Comment (Perplexer-HNP) [inline scenes/combat/combat.tscn:1]: do we not want a combat scene anymore? I dont follow why this and the other combat stuff is being removed

File: systems/combat/CombatManager.cs
Comment (Perplexer-HNP) [inline systems/combat/CombatManager.cs:1]: I feel like this is important for combat to function, but unsure if still being used.

File: systems/combat/ICombatant.cs
Comment (Perplexer-HNP) [inline systems/combat/ICombatant.cs:1]: Would be used by combat manager.

File: systems/combat/Deck.cs
Comment (Perplexer-HNP) [inline systems/combat/Deck.cs:1]: Combat having a deck is probably required, unless this functionallity has been moved.

File: data_resources/card_data/CardData.cs
Comment (Perplexer-HNP) [inline data_resources/card_data/CardData.cs:1]: DamageDice is being used, why is it being deleted here ?

File: scenes/shared/Card/card.tscn
Comment (Perplexer-HNP) [inline scenes/shared/Card/card.tscn:1]: Might be outdated? But feel like this is probably important.

**Review by Perplexer-HNP — COMMENTED** [review]
Removes a lot of relevant files and code, unsure on reasoning. Some may be outdated, and I could have missed things, but I know for sure that atleast some of the combat stuff is relevant / current.

**Review by Perplexer-HNP — APPROVED** [review]
Cleaned up dead files / unused filler assets / tile map.

Seems good.

**Pull request closed without merging**